### PR TITLE
Fix #6893 : UI improvements and bug fixes 1

### LINKED ISSUE #6893: UI Improvements_2
- [x]  Manage button styling on team detail page is not consistent 
Screenshot 2022-08-24 at 8 49 44 AM

- [x] Replace Delete Icon with Remove icon 
Remove
Screenshot 2022-08-24 at 8 51 39 AM

- [x] Owner's link on teams page not working, either not re-directing or giving a 404 error
- [x] Add a breadcrumb link to the main user listing page on create user form
Screenshot 2022-08-24 at 8 55 55 AM
 

- [x] Can we keep the styiling in image 1 same as it is in image 2
Image 1
Screenshot 2022-08-24 at 8 59 53 AM
Image 2
Screenshot 2022-08-24 at 9 01 13 AM

- [x] "Database Service" click is redirecting to the old Service page
Screenshot 2022-08-24 at 9 01 57 AM

- [x] mlflow logo is stretched over here
Screenshot 2022-08-24 at 9 04 19 AM

- [x] Service type (Breadcrumb) click on add service form is redirecting to the old Service page

- [x] Can we list these chips in 1 line below the filter option instead of filtering it inside on "Webhook" and "Slack" page
Screenshot 2022-08-24 at 9 09 28 AM

- [x]  Explore Page Left column isn't using full height on applying filter
Screenshot 2022-08-24 at 9 29 34 AM

- [x] Reduce table column height on user page and admins page

- [x] Edit owner dropdown position should be below the icon
Screenshot 2022-08-25 at 6 11 21 PM

- [x] Manage button styling on glossary page is not consistent with other pages
Screenshot 2022-08-25 at 6 36 25 PM

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes :

I worked on the UI Improvements

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">

- [x] Changed service icon for MlModel service.
Screenshot 2022-08-25 at 11 33 53 PM

- [x] Changed No data placeholders on several tabs to make it consistent with others
Screenshot 2022-08-25 at 11 34 37 PM
Screenshot 2022-08-25 at 11 34 46 PM
Screenshot 2022-08-25 at 11 34 53 PM

- [x] Made left panel height fixed and content scrollable

https://user-images.githubusercontent.com/51777795/186737676-e6d74a04-9016-49e7-a53d-90c64faba7fb.mov

</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less [hunk 1]
```diff
@@ -74,3 +72,16 @@
...
+
+.global-settings-left-panel-container {
+  height: calc(100vh - 90px);
+  border: 1px solid @left-panel-border-color;
+  border-radius: 4px;
+  box-shadow: 1px 1px 8px @box-shadow-color;
+  overflow-y: scroll;
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+}
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less:84]: We have css for this panel under this class: `global-setting-left-panel`

File: openmetadata-ui/src/main/resources/ui/src/pages/tags/TagsPage.style.less [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+.tags-page-left-panel {
+  height: calc(100vh - 90px);
+  overflow-y: scroll;
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+}
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/tags/TagsPage.style.less:19]: Instead specific class for left panel can you make some generic style for left panel?

File: openmetadata-ui/src/main/resources/ui/src/styles/x-master.css [hunk 3]
```diff
@@ -786,7 +786,7 @@ body .profiler-graph .recharts-active-dot circle {
   position: fixed;
   overflow-y: scroll;
   width: 304px;
-  height: calc(100vh - 90px);
+  height: calc(100vh - 80px);
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/styles/x-master.css:789]: I won't recommend this. Let's discuss this offline

**Review by ShaileshParmar11 — APPROVED** [review]

**Review by chirag-madlani — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less (hunk 1 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less]: made a global styling for class left-panel-container and removed this

File: openmetadata-ui/src/main/resources/ui/src/styles/x-master.css (hunk 3 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/styles/x-master.css]: Made changes as discussed

File: openmetadata-ui/src/main/resources/ui/src/pages/tags/TagsPage.style.less (hunk 2 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/pages/tags/TagsPage.style.less]: made generic class left-panel-container for styling left panel

**Pull request merged**